hack/local-garden/run-kube-etcd

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+LABEL=${1:- local-garden}
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+
+IMAGE=quay.io/coreos/etcd:v3.3.10
+PORT=12380
+ETCDPORT=12379
+CLUSTERNAME=gardener-etcd
+
+# Change this value to the desired storage location
+ETCD_DATA_DIR="$PWD/dev/gardener-etcd/kubernetes"
+
+
+PORTS="-p $PORT:$PORT -p $ETCDPORT:$ETCDPORT"
+MOUNTS="-v $SCRIPTPATH/certificates/certs:/certs -v $SCRIPTPATH/certificates/keys:/keys -v $ETCD_DATA_DIR:/etcd-data"
+
+
+echo "Starting gardener-dev kube-etcd cluster..!"
+docker run -d --name etcd -l $LABEL --network gardener-dev --rm  $MOUNTS $IMAGE etcd --name $CLUSTERNAME  \
+ --listen-client-urls "https://0.0.0.0:${ETCDPORT}" \
+ --advertise-client-urls "https://0.0.0.0:${ETCDPORT}" \
+ --listen-peer-urls "https://0.0.0.0:${PORT}" \
+ --trusted-ca-file "certs/ca.crt" \
+ --peer-trusted-ca-file "certs/ca.crt" \
```

Comment:
Do we really need all that `peer` settings for such a simple setup?

Reply:
now not really, but in the future with multiple `etcds` it can be useful to have (as for now, it won't be really used).